services/merak-topo/handler/handler.go

```diff
@@ -203,24 +293,22 @@ func UpdateComputenodeInfo(client *kubernetes.Clientset, topo_id string, returnM
 	// log.Printf("updatecomputenode:=========Update mac addresses ===========")
 
 	for _, node := range topo.Vnodes {
-		var cnode pb_common.InternalComputeInfo
 
 		if strings.Contains(node.Name, "vhost") {
 
-			res, err := client.CoreV1().Pods("default").Get(Ctx, node.Name, metav1.GetOptions{})
+			var cnode database.ComputeNode
 
-			if err != nil {
-				cnode.Name = node.Name
-				cnode.Id = node.Id
-				cnode.DatapathIp = strings.Split(node.Nics[len(node.Nics)-1].Ip, "/")[0]
-				cnode.Veth = node.Nics[len(node.Nics)-1].Intf
-				cnode.OperationType = pb_common.OperationType_INFO
-				cnode.Status = pb_common.Status_ERROR
+			res, err := k8client.CoreV1().Pods("default").Get(Ctx, node.Name, metav1.GetOptions{})
 
-				returnMessage.ComputeNodes = append(returnMessage.ComputeNodes, &cnode)
-				return fmt.Errorf("get pod error %s", err)
+			if err != nil {
+				// cnode.Name = node.Name
+				// cnode.Id = node.Id
+				// cnode.DatapathIp = strings.Split(node.Nics[len(node.Nics)-1].Ip, "/")[0]
+				// cnode.Veth = node.Nics[len(node.Nics)-1].Intf
+				// cnode.OperationType = database.OPERATION_INFO
+				// cnode.Status = database.STATUS_DEPLOYING
```

Comment:
@zzengcs  please remove it if you don't need it.

Reply:
resolved